astropy/utils/masked/core.py

```diff
@@ -241,7 +241,7 @@ def unmasked(self):
 
         See Also
         --------
-        filled
+        astropy.utils.masked.Masked.filled
```

Comment:
According to numpydoc SA errors (at the bottom of https://numpydoc.readthedocs.io/en/latest/validation.html), I think this and below should also have an explanation. In #11597 I'm trying to turn on this validation and correct any docs.